reviews/graduation-fluentd.md

```diff
@@ -0,0 +1,74 @@
+# Fluentd Graduation Application
+
+[Fluentd](https://www.fluentd.org) project has been in incubation for one year and a half. The following application links to the required information to become a graduated project plus some additional information about it growth.
+
+### Fluentd Adoption and Ecosystem
+
+In terms of adoption, It's hard to measure the real numbers of deployments of Fluentd, but based in the public statistics considering only our own Docker images, in the last year  we see more than <u>15 million of pulls</u> .
```

Comment:
> last year  we see more than <u>15 million of pulls</u> 

I'll nit a little. After year there are two spaces and for grammar the _of_ between million and pulls can be removed. This is just a grammar nit rather than something impacting content.

Reply:
thanks. fixed.